### PR TITLE
add a few useful methods to AtomicInterval

### PR DESCRIPTION
**Pull request by jeffwashington**
#### Summary of Changes

Fixes #

### CODE REVIEW

**Comment by jeffwashington** [comment]
Let's consider the orderings here.

File: sdk/src/timing.rs [hunk 1]
```diff
@@ -80,6 +80,18 @@ impl AtomicInterval {
                 == Ok(last)
             && !(skip_first && last == 0)
     }
+
+    /// return ms elapsed since the last time the time was set
+    pub fn elapsed_ms(&self) -> u64 {
+        let now = timestamp();
+        let last = self.last_update.load(Ordering::Relaxed);
+        now.saturating_sub(last) // wrapping somehow?
+    }
```
Comment (brooksprumo) [inline sdk/src/timing.rs:89]: Can `should_update_ext()` be implemented using `elapsed_ms()` for its first 3ish lines?

**Review by brooksprumo — COMMENTED** [review]
Looks good; had one suggestion

**Review by brooksprumo — DISMISSED** [review]

File: sdk/src/timing.rs (hunk 1 above)
Comment (brooksprumo) [reply sdk/src/timing.rs:89]: Nevermind!

**Pull request merged**